build: update to latest stable Chromium 83.0.4103 (#37427)
angular/angular: dev-infra/browsers/README.md

> # Chrome browser versions for testing and angular

Reviewer: ```suggestion
# Chrome browser versions for testing and Angular
```

Author: Done

> # Chrome browser versions for testing in angular

Reviewer: ```suggestion
# Browser configuration and versioning for testing of Angular
```

Author: 👍

> The version of chrome used in tests is determined in a few places.

Reviewer: ```suggestion
Within the Angular monorepo, we use Chrome to perform most of the local testing, and rely on Sauce Labs and BrowserStack to do cross-browser testing on our CI.

The version of Chrome used in tests within this monorepo is configured and controlled via `rules_webtesting` and `pupeteer`. We manually keep the configuration of these two tools in sync to create a consistent testing environment across unit, e2e, and integration tests.
```

Author: 👍

## rules_webtesting

> Bazel `karma_web_test_suit`e and `protractor_web_test_suite` targets will use Chromium provisioned by rules_webtesting. The version of chrome used is specified in the `dev-infra/browsers/browser_repositories.bzl` file. Its a bit of a process to upgrade the version in that file:

Reviewer: ```suggestion
Bazel `karma_web_test_suite` and `protractor_web_test_suite` targets will use Chromium provisioned by rules_webtesting. The version of chrome used is specified in the `dev-infra/browsers/browser_repositories.bzl` file. The process of updating the Chrome version in that file consists of several steps:
```

Author: 👍
